### ISSUE TITLE
enrichment moving parentheses out of table

### ISSUE THREAD

**Issue by pkra** [zorkow/speech-rule-engine#616]
Minimal example:

```tex
\begin{equation}
  (x) \tag{1}
\end{equation} 
```

results in something like

```
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
  <mrow data-semantic-type="vector" data-semantic-role="squarematrix" data-semantic-id="8" data-semantic-children="7" data-semantic-content="2,4" data-semantic-speech="Start 1 By 1 Matrix 1st Row  with Label left parenthesis 1 right parenthesis EndLabel x EndMatrix">
    <mo stretchy="false" data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="8">(</mo>
    <mtable displaystyle="true">
      <mlabeledtr data-semantic-type="line" data-semantic-role="squarematrix" data-semantic-id="7" data-semantic-children="3" data-semantic-content="1" data-semantic-parent="8">
        <mtd id="mjx-eqn:1" data-semantic-type="cell" data-semantic-role="label" data-semantic-id="1" data-semantic-children="0" data-semantic-parent="7">
          <mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-id="0" data-semantic-parent="1">(1)</mtext>
        </mtd>
        <mtd>
          <mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7">x</mi>
        </mtd>
      </mlabeledtr>
    </mtable>
    <mo stretchy="false" data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="8">)</mo>
  </mrow>
</math>
```

Reproducible on mathjax.org/#demo and with 4.0.0-beta.2.

**Comment by pkra** [comment]
Full real world example:

```tex
\begin{equation}
  ( L_{1}, \ldots , L_j, \sigma _{-r}, \ldots , \sigma _{-1}, L'_{1}, \ldots , L'_k, \sigma _{0}^\phi , \sigma _{1}^\phi , \ldots , \sigma _\ell ^\phi ) \tag{3.42}
\end{equation}
```

**Comment by pkra** [comment]
Hack to workaround: adding a preceding space, e.g., `\begin{equation}  \ (x) \tag{1} \end{equation} `

**Comment by pkra** [comment]
Resolved in 4.0.0-beta.3
